### PR TITLE
updates IP forwarding docs

### PR DESCRIPTION
**Pull request by stevsmit**
Version(s):
4.14+

Issue:
https://issues.redhat.com/browse/OCPBUGS-23276

Link to docs preview:
[
](https://68684--docspreview.netlify.app/openshift-enterprise/latest/networking/ovn_kubernetes_network_provider/configuring-egress-ips-ovn#nw-egress-ips-multi-nic-considerations_configuring-egress-ips-ovn)

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/nw-egress-ips-about.adoc [hunk 1]
```diff
@@ -188,7 +188,7 @@ For users who want an egress IP and traffic to be routed over a particular inter
 
 * The Egress IP must be IPv4. IPv6 is currently unsupported.
 
-* IP forwarding must be enabled for the network interface. To enable IP forwarding, you can set `ipForwarding: Global` in your `config.yaml` file, for example:
+* IP forwarding must be enabled for the network interface. To enable IP forwarding, you can set `ipForwarding: Global` in your Cluster Network Operator configuration object. For example:
```
Comment (huiran0826) [inline modules/nw-egress-ips-about.adoc:191]: It seems still not cleared how to update it. Here is where to update this add value. 
oc edit network.operator
# ....
spec:
  clusterNetwork:
  - cidr: 10.128.0.0/14
    hostPrefix: 23
  defaultNetwork:
    ovnKubernetesConfig:
      gatewayConfig:
        ipForwarding: Global
#....

**Review by skopacz1 — COMMENTED** [review]
Left a few clarifying questions, otherwise looks good to me!

**Pull request merged**